api/base.cfg

```diff
@@ -1,6 +1,6 @@
 [buildout]
 index = https://pypi.org/simple/
-parts = instance plonesite robot-server
```

Comment:
@sneridagh why don't we create a plonesite any longer?

Reply:
fixed.